srcpkgs/itch/template

```diff
@@ -0,0 +1,61 @@
+# Template file for 'itch'
+pkgname=itch
+version=1.18.0
+revision=1
+archs="i686 x86_64"
+build_style=go
+hostmakedepends="wget"
+makedepends="curl"
+short_desc="Itch.io indie games client for Linux"
+maintainer="MarcoAPC <marcoaureliopc@gmail.com>"
+license="MIT"
+homepage="https://github.com/itchio/itch"
+
+case "$XBPS_TARGET_MACHINE" in
+	x86_64)
+		_dist_arch="amd64"
+		_filename=itch-setup
```

Comment:
What is `itch-setup` and why does or have to be fetched as a binary?

Reply:
I tried to download the binary to be faster in building the package, I will modify the template to compile the source.